server/routes.js

```diff
@@ -317,6 +317,7 @@ exports.isomorphicRoutes = function isomorphicRoutes(
     redirectToLowercaseSlugs = false,
     shouldEncodeAmpUri,
     sMaxAge = 900,
+    lowfidelitySvg = "",
```

Comment:
It doesn't have to be an SVG, can be any string so maybe we can use a more appropriate name like `appLoadingPlaceholder`.

Also another point is that we must document this parameter just above this function along with the other documentation for other parameters.